src/unix.c

```diff
@@ -43,6 +43,13 @@ static int connUnixAddr(connection *conn, char *ip, size_t ip_len, int *port, in
     return connectionTypeTcp()->addr(conn, ip, ip_len, port, remote);
 }
 
+static int connUnixIsLocal(connection *conn)
+{
+    UNUSED(conn);
+
+    return 1; /* of cause, Unix socket is always local connection */
```

Comment:
had a typo, but let's drop that part of the line to simplify.
```suggestion
    return 1; /* Unix socket is always local connection */
```

Reply:
OK